README-FIPS.md

```diff
@@ -31,9 +31,9 @@ the `enable-fips` option.
 Installing the FIPS provider
 ============================
 
-You can build the OpenSSL source code and the FIPS provider by doing the following
-if you only use the FIPS validated source code. If you want to use a validated FIPS
-module with the latest release see the next section.
+You can build the OpenSSL source code with a validated FIPS provider by doing the
+following if you only use the FIPS validated source code. If you want to use a
```

Comment:
s/if you only use/, only if you are using/

Reply:
CHanged the format